platform/pulp/plugin/changeset/main.py

```diff
@@ -0,0 +1,200 @@
+from collections import Sized, Iterable
```

Comment:
I'll update the import to be on safe side.  However, It would be better if the author kept them permanently (and not just for backwards compat) in the ``collections`` package interface.  I really don't understand why the python community makes such poor choices.

https://github.com/python/cpython/blob/3.6/Lib/collections/__init__.py#L22